ceph/utils.py

```diff
@@ -50,7 +50,11 @@ def cleanup_ibmc_ceph_nodes(ibm_cred, pattern):
         log.warning("Failed to retrieve instances")
         return 1
 
-    instances = [i for i in resp.get_result()["instances"] if pattern in i["name"]]
+    instances = [
+        i
+        for i in resp.get_result()["instances"]
+        if pattern in i["name"] and not i["name"].startswith("dnd")
```

Comment:
We shouldn't be doing this check. The infra or critical systems must be moved to a separate project which CI has no control. 

Adding such checks without documentation is a maintenance nightmare.